aws-environment-param-sync/src/index.js

```diff
@@ -0,0 +1,177 @@
+const core = require('@actions/core');
+const { SSM } = require('aws-sdk');
+const { Octokit } = require("@octokit/core");
+
+const getParameter = async ({
+     path,
+  }) => {
+  const ssm = new SSM();
+  const parameters = {};
+  let nextToken = undefined;
+
+  do {
+    const { Parameters, NextToken } = await ssm
+      .getParametersByPath({
+        Path: path,
+        Recursive: true,
+        WithDecryption: false,
+        NextToken: nextToken,
+      })
+      .promise();
+
+    Parameters.forEach((parameter) => {
+      const name = formatParameterName(parameter.Name);
+      const value = parameter.Value.trim();
+
+      core.info('Param: ' + parameter.Name + ' Name: ' + name + ' Value: ' + value);
```

Comment:
Can use string templates. Few other places if you want.
```suggestion
      core.info(`Param: ${parameter.Name} Name: ${name} Value: ${value}`);
```

Reply:
Nice, done